src/officehours_api/serializers.py

```diff
@@ -257,3 +257,8 @@ def validate_queue(self, queue):
         ):
             raise serializers.ValidationError(f'Queue {queue} is closed.')
         return queue
+
+    def validate(self, attrs):
+        if attrs["assignee"] and attrs["assignee"] not in attrs["queue"].hosts.all():
```

Comment:
I think this is breaking all host assignments, because the patch `api` functions you have aren't specifying `host`. I think you may need to go get the `queue` yourself based on the meeting id.

Reply:
I discovered it's already there as `self.instance`, so I changed it to use that if it's present, which makes it throw the error as expected and also works for the happy paths. 

As I worked on this, it occurred to me that we should spend some time auditing our API endpoints, removing any that aren't needed for the UI and making sure you can't do things like assign foreign keys in a way that doesn't jive with business logic. (#289)

Reply:
Nice that you can get `queue` via `instance`. Change looks good.